terraform/modules/statsd/variables.tf

```diff
@@ -40,3 +40,9 @@ variable "task_role_arn" {
 variable "vpc_id" {
   type = string
 }
+
+variable "desired_count" {
+  description = "Desired count of Application instances"
```

Comment:
I'd recommend omitting the description in this case as it doesn't really add anything.

Reply:
sorry missed this comment - will keep that in mind for next